iotlab_controller/riot.py

```diff
@@ -41,23 +41,19 @@ def path(self):
         else:
             return self.flashfile
 
-    def build(self, build_env=None, threads=1):
+    def run(self, build_env, cmd):
```

Comment:
Make this private please.

```suggestion
    def _run(self, build_env, cmd):
```

There is no meaning for this outside of the context of the public methods below.

Reply:
ups, that was simply because of my newby python skills, did never look up how to declare functions private in python... Will of course fix.

Reply:
It's more [by convention than real declaration](https://docs.python.org/3/tutorial/classes.html#private-variables), but linters will look for that so its good to keep this convention ;-).